File: apps/console/BasketballConsole/BasketballConsole/Models/CosmosDB/Participation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NBA.Models.CosmosDB
{
    internal class Participation
    {
        public string Id { get; set; }
        public string GameId { get; set; }
        public int PlayerId { get; set; }
        public string PlayerName { get; set; }
        public string TeamName { get; set; }
        public List<GamePlay> Plays { get; set; } = [];
    }
}

File: Vue-Nuxt with Java-MongoDB/client/nba-statistics/models/Participation.ts
import { GamePlay } from './GamePlay';

export class Participation {
    participationId: string;
    gameId: string;
    playerId: string;
    plays: GamePlay[];

    constructor(
        participationId: string,
        gameId: string,
        playerId: string,
        plays: GamePlay[]
    ) {
        this.participationId = participationId;
        this.gameId = gameId;
        this.playerId = playerId;
        this.plays = plays;
    }
}
